core/src/androidMain/kotlin/io/tolgee/storage/TolgeeStorageProviderAndroid.kt

```diff
@@ -0,0 +1,35 @@
+package io.tolgee.storage
+
+import android.content.Context
+import java.io.File
+
+class TolgeeStorageProviderAndroid(
```

Comment:
Following extension functions are available which use `NIO` on supported devices and won't cause crashes:

- `File?.existsSafely` checks if file exists
- `File?.existsRSafely` checks if file exists and we have read permission
- `File?.existsRWSafely` checks if file exists and we have read/write permissions
- `File.canReadSafely` checks whether we have read permission
- `File.canWriteSafely` checks whether we have write permission
- `File.mkdirsSafely` create this and parent directories